src/main/java/com/elixr/poc/util/PropertiesUtil.java

```diff
@@ -0,0 +1,39 @@
+package com.elixr.poc.util;
+
+import lombok.extern.log4j.Log4j2;
+import org.apache.catalina.core.Constants;
+
+import java.util.Properties;
+
+@Log4j2
+public class PropertiesUtil {
+    private static final String PATH = "/messages.properties";
```

Comment:
Check how properties files can be loaded using configuration class in spring boot.https://www.baeldung.com/configuration-properties-in-spring-boot

Reply:
Updated